Source/WebKit/UIProcess/Cocoa/VideoPresentationManagerProxy.h

```diff
@@ -114,6 +114,10 @@ class VideoPresentationModelContext final
     void didExitFullscreen() final;
     void didCleanupFullscreen() final;
     void fullscreenMayReturnToInline() final;
+#if PLATFORM(VISION)
```

Comment:
```suggestion
#if ENABLE(LINEAR_MEDIA_PLAYER)
```

As that’s when those methods are defined